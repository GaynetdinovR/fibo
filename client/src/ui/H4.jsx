import styles from "../styles/Ui.module.sass";
import classNames from "classnames";

const H4 = ({ children, className }) => {
	return (
		<h4 className={classNames(className, styles.h4)}>
			{children}
		</h4>
	);
};

export default H4;