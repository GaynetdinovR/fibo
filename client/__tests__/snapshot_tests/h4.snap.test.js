import renderer from "react-test-renderer";
import { it, expect } from "@jest/globals";
import H4 from "../../src/ui/H4.jsx";

it("renders correctly", () => {
	const tree = renderer.create(<H4>H4</H4>).toJSON();
	expect(tree).toMatchSnapshot();
});
