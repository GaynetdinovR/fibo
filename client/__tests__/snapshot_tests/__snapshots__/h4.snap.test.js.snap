// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<h4
  className="h4"
>
  H4
</h4>
`;
